source/features/batch-mark-files-as-viewed.tsx

```diff
@@ -63,6 +64,11 @@ function onAltClick(event: delegate.Event<MouseEvent, HTMLInputElement>): void {
 }
 
 function init(): Deinit[] {
+	// eslint-disable-next-line @typescript-eslint/no-floating-promises
+	elementReady('.js-file').then(file => {
+		previousFile = file;
+	});
```

Comment:
Let's not use `elementReady` when we don't need it.

This can be implemented in a simpler way: in `batchToggle`, check if `previousFile` is undefined. If it is, select the first `.js-file` element.